content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -0,0 +1,281 @@
+---
+title: "Install and Manage MAAS Gateway"
+metaTitle: "Install and Manage MAAS Private Cloud Gateway"
+metaDescription: "Learn how to install and manage the MAAS Private Cloud Gateway in Palette."
+hideToC: false
+fullWidth: false
+---
+
+import Tabs from 'shared/components/ui/Tabs';
+import WarningBox from 'shared/components/WarningBox';
+import InfoBox from 'shared/components/InfoBox';
+import PointsOfInterest from 'shared/components/common/PointOfInterest';
+
+# Overview 
+
+The Private Cloud Gateway (PCG) enables support for private cloud or data center environments. You can set up the cloud gateway as a single-node or three-node cluster based on your requirements for high availability (HA).  
+<br />
+
+<InfoBox>
+
+For production environments, we recommend using three nodes. If you initially set up the gateway with one node, you can resize it at a later time. 
+
+</InfoBox>
+
+
+Palette provides an installer in the form of a Docker container that is temporarily deployed on your laptop, workstation or jump box. You can use the installer on any linux x86-64 system that has a Docker daemon installed and connectivity to Palette and the MAAS identity endpoint. 
+
+<br />
+
+
+<WarningBox>
+
+The installer does not currently work on MacOS.
+
+</WarningBox>
+
+
+## Prerequisites
+
+- Canonical [MAAS installed](https://maas.io/docs/how-to-install-maas), set up, and available in your environment.
+
+
+- A linux computer with a Docker daemon installed and a connection to Palette and the MAAS endpoint. The installer must be invoked on a linux system. We have tested the gateway installation using Ubuntu 20.04 on x86-64.
+
+
+- Private cloud gateway IP requirements: <br /><br /> 
+    
+    - One IP address for a single-node gateway or three IP addresses for a three-node gateway.
+    - One IP address for a Kubernetes control plane.
+
+
+- Sufficient IPs for application workload services, such as Load Balancer services.
+
+<WarningBox>
+
+By default, the MAAS Kubernetes pack uses 192.168.0.0/16. Ensure that the Pod classless inter-domain routing (CIDR) range for any clusters you deploy after setting up the PCG do not overlap with the network used by the bare metal machines that MAAS manages.
+
+</WarningBox>
+
+- Each node in the PCG cluster requires a machine from MAAS in a ready state, and with the following resources:
+
+    <br />
+
+    - 4 CPUs
+    - 4096 MiB memory
+    - 60 GiB storage<br /><br />
```

Comment:
```suggestion
    - 60 GiB storage
```
I'm not sure why we are using double break lines here,

Reply:
fixed